### PR TITLE
NMS-14171: add docs for event performance data

### PR DESCRIPTION
**Pull request by freddy61025**
https://issues.opennms.org/browse/NMS-14171

### CODE REVIEW

File: docs/modules/operation/pages/events/perf-data.adoc [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+ --vertical-label="Bytes per second" \
+ DEF:octIn={rrd1}:bytesIn:AVERAGE \
+ AREA:octIn#73d216: \
+ LINE1:octIn#4e9a06:"In " \
+ GPRINT:octIn:AVERAGE:"Avg  \\: %8.2lf %s"
+```
+
+#### create resource-types if you are using generic type
+If your time series data type is non nodeSnmp / interfaceSnmp. You will need to create a resource type file in etc/resource-types.d.
+The most important thing is to match the name with evenconf's resourceType. There is also ${instance} variable available.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/events/perf-data.adoc:155]:
```suggestion
Make sure you match the name with evenconf's resourceType. 
There is also ${instance} variable available.
```
why is the {instance} variable important?

File: docs/modules/operation/pages/events/perf-data.adoc [hunk 2]
```diff
@@ -0,0 +1,168 @@
+= Performance data
+
+This feature initially is triggered by customer who want to persist SNMP traps in time series database. Since traps are already persist in OpenNMS an events form. Moreover, trap output is not the same as SNMP. It always embeds values into a string. Events support named regex extraction, and it is a good starting point for persisting extracted parameters into time series.
+== Flow of persisting
+- event matching with eventconf and check collection information exists
```
[the remark this answers is not in the document]
Comment (freddy61025) [reply docs/modules/operation/pages/events/perf-data.adoc:5]: It refer to trap's output data.

File: docs/modules/operation/pages/events/perf-data.adoc [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+- It will try to lookup parameters stated in config against event's parameters
```
[the remark this answers is not in the document]
Comment (freddy61025) [reply docs/modules/operation/pages/events/perf-data.adoc:6]: "It" is the feature, or you could say OpenNMS.

File: docs/modules/operation/pages/events/perf-data.adoc (hunk 1 above)
Comment (freddy61025) [reply docs/modules/operation/pages/events/perf-data.adoc:155]: for example you are receiving a network switch traps. {instance} can be port name. Without a proper name, you won't know what is that events bind to.

File: docs/modules/operation/pages/events/perf-data.adoc [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+- if paramValue in eventconf exists, it will convert the mating key into numeric values.
```
[the remark this answers is not in the document]
Comment (freddy61025) [reply docs/modules/operation/pages/events/perf-data.adoc:7]: It is a typo.

File: docs/modules/operation/pages/events/perf-data.adoc [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+- Based on the collection setting, it will persist data into nodeSnmp, interfaceSnmp or resource type.
```
[the remark this answers is not in the document]
Comment (freddy61025) [reply docs/modules/operation/pages/events/perf-data.adoc]: It means the feature / OpenNMS

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
A few minor typos I missed the first time. I also added information from your comments. I hope I got it right.

**Pull request merged**